modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/rest/resource/SW360Attributes.java

```diff
@@ -20,50 +20,14 @@ private SW360Attributes() {}
     public static final String PROJECT_RELEASES_TRANSITIVE ="transitive";
     public static final String COMPONENT_SEARCH_BY_NAME = "name";
 
-
-    // Attributes of Sw360Project
-    public static final String PROJECT_ID = "id";
-    public static final String PROJECT_TYPE = "type";
-    public static final String PROJECT_NAME = "name";
-    public static final String PROJECT_VERSION = "version";
-    public static final String PROJECT_PROJECT_TYPE = "projectType";
-    public static final String PROJECT_DESCRIPTION = "description";
-    public static final String PROJECT_EXTERNAL_IDS = "externalIds";
-    public static final String PROJECT_CREATED_ON= "createdOn";
-    public static final String PROJECT_BUSINESS_UNIT = "businessUnit";
-    public static final String PROJECT_CLEARING_TEAM= "clearingTeam";
-    public static final String PROJECT_VISIBILITY = "visbility";
-    public static final String PROJECT_RELEASE_ID_TO_USAGE = "releaseIdToUsage";
-
-    // Attributes of Sw360Component
-    public static final String COMPONENT_ID = "id";
-    public static final String COMPONENT_COMPONENT_NAME = "name";
-    public static final String COMPONENT_COMPONENT_TYPE = "componentType";
-    public static final String COMPONENT_TYPE = "type";
-    public static final String COMPONENT_CREATED_ON = "createdOn";
-    public static final String COMPONENT_CATEGORIES = "categories";
-    public static final String COMPONENT_HOMEPAGE = "homepage";
-
-    // Attributes of Sw360Release
-    public static final String RELEASE_ID = "id";
-    public static final String RELEASE_COMPONENT_ID = "componentId";
-    public static final String RELEASE_NAME = "name";
-    public static final String RELEASE_VERSION = "version";
-    public static final String RELEASE_CPE_ID = "cpeId";
-    public static final String RELEASE_SOURCES = "downloadurl";
-    public static final String RELEASE_MAIN_LICENSE_IDS = "mainLicenseIds";
-    public static final String RELEASE_EXTERNAL_IDS = "externalIds";
-
     // Attributes of Sw360Attachment
     public static final String ATTACHMENT_ATTACHMENT_TYPE = "attachmentType";
     public static final String ATTACHMENT_CHECK_STATUS = "checkStatus";
 
     // Attributes of Sw360License
-    public static final String LICENSE_TYPE = "type";
     public static final String LICENSE_TEXT = "text";
```

Comment:
Why do you keep still these license attributes?

Reply:
those are currently still in use. since I have not changed the way Licenses get serialized, since this changes based on whether or not the `SW360License` has a full name or not. I didn't find a way to implement this logic in JsonSerialization. Suggestions are welcome